tests/pipeline/test_buyback_auth.py

```diff
@@ -34,16 +34,16 @@
     BlazeShareBuybackAuthorizationsLoader,
 )
 from zipline.pipeline.loaders.utils import (
-    get_values_for_date_ranges,
     zip_with_floats,
     zip_with_dates
 )
 from zipline.testing.fixtures import (
     WithPipelineEventDataLoader, ZiplineTestCase
 )
 
-date_intervals = [[None, '2014-01-04'], ['2014-01-05', '2014-01-09'],
-                  ['2014-01-10', None]]
+date_intervals = [
```

Comment:
Why is this nested one level deeper now? The docs for the `get_sids_to_frames` seemed to indicate that it was just an iterable of date ranges. This appears to be an iter[iter[date ranges]].